x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/prebuilt_rule_import.md

```diff
@@ -91,195 +119,429 @@ User stories:
 
-**Automation**: 1 integration test.
+**Automation**: 1 API integration test.
 
 ```Gherkin
-Given the import payload contains a prebuilt rule without a rule_id field
+Given the import payload contains a custom rule
+And its rule_id does NOT match any rule assets from the installed package
+And this rule is already created
 When the user imports the rule
-Then the import should be rejected with a message "rule_id field is required"
+Then the rule should be updated
+And the updated rule should be custom
+And the updated rule's parameters should match the import payload
 ```
 
-#### Scenario: Importing a prebuilt rule with a matching `rule_id` but missing a `version` field
+### Importing multiple rules in bulk
+
+#### **Scenario: Importing a mixture of new prebuilt and custom rules**
+
+This scenario is a "smoke test" for all the user stories from the [Product requirements](#product-requirements) section.
 
-**Automation**: 1 integration test.
+**Automation**: 1 API integration test, 1 e2e test.
 
 ```Gherkin
-Given the import payload contains a prebuilt rule without a version field
-And its rule_id matches one or a few rule assets from the installed package
+Given the import payload contains prebuilt non-customized, prebuilt customized, and custom rules
+And the prebuilt rules have a base version in the installed package
+And the custom rules' rule_id does NOT match any rule assets from the installed package
+And the rules are not installed or created yet
+When the user imports these rules
+Then the rules should be created
+And the created rules should be correctly identified as prebuilt or custom
+And the created rules' is_customized field should be correctly calculated
+And the created rules' parameters should match the import payload
+```
+
+#### **Scenario: Importing a mixture of prebuilt and custom rules on top of existing rules**
+
+This scenario is a "smoke test" for all the user stories from the [Product requirements](#product-requirements) section.
+
+**Automation**: 1 API integration test, 1 e2e test.
+
+```Gherkin
+Given the import payload contains prebuilt non-customized, prebuilt customized, and custom rules
+And the prebuilt rules have a base version in the installed package
+And the custom rules' rule_id does NOT match any rule assets from the installed package
+And the rules are already installed or created
+When the user imports these rules
+Then the rules should be updated
+And the updated rules should be correctly identified as prebuilt or custom
+And the updated rules' is_customized field should be correctly calculated
+And the updated rules' parameters should match the import payload
+```
+
+### Importing prebuilt rules when the package is not installed
+
+#### **Scenario: Importing new prebuilt rules when the package is not installed**
+
+**Automation**: 1 API integration test.
+
+```Gherkin
+Given the import payload contains two prebuilt rules (non-customized + customized)
```

Comment:
`two prebuilt rules (non-customized + customized)` - are rules combined to squash two scenarios in one?

Reply:
@maximpn Yes, since all the steps in these two scenarios are the same (except the import payload), we can squash them into a single one, and automate this in a single test. It is also logically cohesive: the two scenarios are about the same thing which is `Importing new prebuilt rules when the package is not installed`.